flink-core/src/test/java/org/apache/flink/util/concurrent/ManuallyTriggeredScheduledExecutor.java

```diff
@@ -94,6 +94,11 @@ public Collection<ScheduledFuture<?>> getActiveNonPeriodicScheduledTask() {
         return execService.getActiveNonPeriodicScheduledTask();
     }
 
+    public Collection<ScheduledFuture<?>> getActiveNonPeriodicScheduledRunnable(
```

Comment:
Should we replace `Runnable` with `Task` for consistency with the existing method names?